package.json

```diff
@@ -10,7 +10,7 @@
         "audit-production": "npm audit --production",
         "compile": "tsc --build",
         "clear-dist": "rimraf */dist",
-        "build": "cd fortigate-autoscale && npm install && cd .. && npm run clear-dist && npm run compile",
```

Comment:
pretty sure this is required?

Reply:
> pretty sure this is required?

do you mean cd to the core and run npm install inside?